### PR TITLE
BB2-3179: Update developer documentation for expired data grant

### PR DESCRIPTION
**Pull request by stiwarisemanticbits**
**JIRA Ticket:**
[BB2-3179](https://jira.cms.gov/browse/BB2-3179)

**User Story or Bug Summary:**

### What Does This PR Do?
Update developer documentation for expired data grant for application. This PR lists error which we get while refreshing token with expired data grant

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

* TODO

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? **Yes** or **No**.
    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

* CMSgov/some_repo#42

### Submitter Checklist

I have gone through and verified that...:

* [ ] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [ ] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [ ] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [ ] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [ ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [ ] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [ ] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: _pages/developers.md [hunk 1]
```diff
@@ -459,6 +459,20 @@ Response (unsuccessful with 401 status code):
 ~~~
 If you receive this message, double-check that the request looks correct. If everything looks correct, email [bluebuttonapi@cms.hhs.gov](mailto:bluebuttonapi@cms.hhs.gov), and the Blue Button 2.0 API team can help troubleshoot.
 
+##### Expired Data Access Gramt
+
+If the authorization for accessing user data by an application has expired, the corresponding access token will not be able to be refreshed. Attempts to refresh that token will result in the following error message:
```
Comment (loganbertram) [inline _pages/developers.md:464]: Two quick notes. First, "Grant" is misspelled "Gramt" in the title. Second, did the text here come from somewhere or does it have or need product review? I might reword the first bit to "If an application's authorization for accessing user data has expired..." That's just a suggestion though.

**Review by loganbertram — CHANGES_REQUESTED** [review]
I left a comment on a typo and a suggestion for rewording one sentence. Everything else looks great!

**Review by loganbertram — APPROVED** [review]

File: _pages/developers.md [hunk 2]
```diff
@@ -459,6 +459,20 @@ Response (unsuccessful with 401 status code):
...
+
+Response (unsuccessful with 400 status code):
+```
+{
+  "status_code": 400,
+  "error": "invalid_grant",
+  "error_description": "The authorization for accessing user data has expired. To refresh Medicare data, the end user must re-authenticate and consent to data sharing."
+}
```
Comment (jimmyfagan) [inline _pages/developers.md:472]: This content is good, but it's inconsistent with the other examples in this section. Could you update the above examples so there is more consistency in the included information (i.e. include error descriptions in the output for the other cases too)?

Comment (jimmyfagan) [reply _pages/developers.md:472]: Discussed offline, those other cases are accurate, just happens that those other scenarios do not have an error description and status code included in the response.

File: _pages/developers.md [hunk 3]
```diff
@@ -478,8 +492,8 @@ POST to the BB2.0 API `/expire_authenticated_user` endpoint with the following p
 ##### cURL command
 ~~~
 curl -X POST "https://sandbox.bluebutton.cms.gov/v2/o/expire_authenticated_user/-20140000000001/" \
--u "swBu7LWsCnIRfu530qnfPw1y5vMmER3lAM2L6rq2:\<client_secret\>" \
--H “Content-Length: 0”
+-u "<client_id>:<client_secret>" \
+-H “Content-Length: 0"
```
Comment (jimmyfagan) [inline _pages/developers.md:496]:
```suggestion
-H "Content-Length: 0"
```

**Review by jimmyfagan — APPROVED** [review]
One last unrelated fix suggested in my comments, but this looks good otherwise, thanks!

**Review by jimmyfagan — APPROVED** [review]

**Pull request merged**